website/docs/r/workflow.html.markdown

```diff
@@ -63,38 +39,86 @@ resource "newrelic_workflow" "foo" {
+For example, we have a workflow with two predicates in the issue filter: 
+- `tag.team EXACTLY_MATCHES my_team` meaning "the issue has a tag called `team` and the tag value is set to 'my_team'" 
+- `labels.policyIds EXACTLY_MATCHES 123` meaning "the issue includes an incident triggered by an alert policy with id = 123"
 
+In this case, an issue would have to **both** have the tag and include an incident triggered by the said policy in order to be processed by the workflow.  
+
+Each `issues_filter` block supports the following arguments:
+
+* `name` - (Required) The name of the filter. The name only serves a cosmetic purpose and can only be seen through Terraform and GraphQL API.
+* `type` - (Required) Type of the filter. Please just set this field to `FILTER`. The field is likely to be deprecated/removed in the near future. 
+* `predicate` (Required) A condition an issue event should satisfy to be processed by the workflow 
+  * `attribute` - (Required) Issue event attribute to check
+  * `operator` - (Required) An operator to use to compare the attribute with the provided `values`. 
+One of: `CONTAINS`, `DOES_NOT_CONTAIN`, `EQUAL`, `DOES_NOT_EQUAL`, `DOES_NOT_EXACTLY_MATCH`, `STARTS_WITH`, `ENDS_WITH`,
+`EXACTLY_MATCHES`, `IS`, `IS_NOT`, `LESS_OR_EQUAL`, `LESS_THAN`, `GREATER_OR_EQUAL`, `GREATER_THAN` (see the note below)
+  * `values` - (Required) The `attribute` must match **any** of the values in this list  
+
+~> **NOTE:** We are currently working on improving documentation around for issue filters' attributes and operators. 
+For now, the easiest way to see which attributes your issue events have and which operators they support is though the 
+New Relic UI. In the future, we are going to provide a comprehensive guide to all attributes and operators.
+
+### Muting Rules
+
+An issue can be either "Fully muted", "Partially muted" or "Not muted" (for more on these states please visit [our official documentation page about them](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications/#workflow-behavior)).
+
+Muting rule defines how to handle issues that are partially or fully muted. 
+
+Possible values:
+* DONT_NOTIFY_FULLY_MUTED_ISSUES - Do not send notifications for fully muted issues, do send notifications for partially muted issues
+* DONT_NOTIFY_FULLY_OR_PARTIALLY_MUTED_ISSUES - Do not send notifications for fully or partially muted issues.
+* NOTIFY_ALL_ISSUES - Always send notifications, no matter whether the issue is muted or not
+
+### Nested `destination` blocks
+
+In order to get notified via a workflow you need two things:
+- A [notification_destination](notification_destination.html) that defines reusable credentials/settings for a 
+notification provider (e.g. webhook's `basic` credentials, Slack's OAuth credentials, PagerDuty API key, etc)
+- A [notification_channel](notification_channel.html) that describes additional notification parameters to be used
```

Comment:
we should use this section to explain that channels must be unique per workflow and that once a workflow is deleted we delete the relevant channels as well.